README.md

```diff
@@ -4,13 +4,25 @@ Lendable Clock Abstraction
 [![Latest Stable Version](https://poser.pugx.org/lendable/clock/v/stable)](https://packagist.org/packages/lendable/clock)
 [![License](https://poser.pugx.org/lendable/clock/license)](https://packagist.org/packages/lendable/clock)
 
-Provides an object oriented interface for retrieving the current time.
+Provides an object-oriented interface for retrieving the current time.
 
-This serves to increase the reliability of time sensitive tests, preventing data fixtures
-from having to be created with relative-to-now timestamps and having potential failures where
-the clock ticks over unexpectedly.
+PHP of course provides `\DateTime`, `\DateTimeImmutable` and `\DateTimeInterface`. Why do we need a clock then? Why not just instantiate where required? 
+
+We can obtain the current time with `$now = new \DateTime()` after all, why do we need a `Clock::now(): \DateTimeImmutable` style API to achieve this?
+
+* Underlying implementation can be swapped out to one more suitable for a test environment.
```

Comment:
```suggestion
We can obtain the current time with `$now = new \DateTime()` after all, why do we need a `Clock::now(): \DateTimeImmutable` style API to obtain the current time?

Depending on a Clock rather than constructing native PHP time objects ad hoc means you can both reason and control time. 

* Underlying implementation can be swapped out to one more suitable for a test environment.
```